webpack.config.js

```diff
@@ -17,7 +17,63 @@ module.exports = {
         })
       },
       {
-        test: /\.js$/
+        test: /\.(eot|svg|ttf|woff|woff2|gif)$/,
+        loader: "file-loader?name=/fonts/[name].[ext]"
+      },
+      {
+        test: /\.css$/,
+        use: ExtractTextPlugin.extract({
+          use: ["css-loader"],
+          fallback: "style-loader"
+        })
+      },
+      {
+        test: /\.js$/,
+        exclude: /(node_modules|bower_components)/,
+        use: [
+          {
+            loader: "babel-loader",
```

Comment:
Lets go over this at some point, I really want to understand every part of this file haha.